.eslintrc.js

```diff
@@ -98,6 +98,7 @@ module.exports = {
     "@typescript-eslint/use-unknown-in-catch-callback-variable": "off",
     "@typescript-eslint/no-explicit-any": "error",
     "@typescript-eslint/explicit-module-boundary-types": "error",
+    "@typescript-eslint/consistent-type-imports": ["warn", { fixStyle: "inline-type-imports" }],
```

Comment:
Why do you need this change for this PR?

Reply:
I was adding it because this part of the larger PR is the one that has most of the type import fixes. As you can see there are a couple of files where the only change is the import type fix change 😊.


I added to automatically add the `type` keyword every time I save the file.